### PR TITLE
Setup: Error if profiling is given but unavailable

### PR DESCRIPTION
**Pull request by morrisonlevi**
### Description

This improves the user experience. Before this change, the user would
not get an error nor a warning, but profiling wouldn't get installed.

### Readiness checklist
- [ ] Changelog has been added to the release document.
- [ ] Tests added for this feature/bug.

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.

### CODE REVIEW

**Comment by morrisonlevi** [comment]
@labbati I looked at the testing infra and it wasn't obvious to me how to add a test to check that profiling to fail to install certain platforms/versions. Can you help me out with that?

**Comment by morrisonlevi** [comment]
CircleCI is on an 8+ hour outage o.O
https://status.circleci.com/

File: dockerfiles/verify_packages/installer/utils.sh [hunk 1]
```diff
@@ -92,5 +92,12 @@ get_php_extension_dir() {
 
 generate_installers() {
     version="${1}"
-    sh $(pwd)/tooling/bin/generate-installers.sh ${version} $(pwd)/build/packages
+    sh "$(pwd)/tooling/bin/generate-installers.sh" "${version}" "$(pwd)/build/packages"
+}
+
+dashed_print() {
```
Comment (labbati) [inline dockerfiles/verify_packages/installer/utils.sh:98]: Good idea

**Review by labbati — APPROVED** [review]
LGTM! Thank @morrisonlevi

**Pull request merged**